elm-package.json

```diff
@@ -12,8 +12,12 @@
         "Parser.Number"
     ],
     "dependencies": {
-        "elm-lang/core": "2.0.1 <= v < 2.0.2",
+        "TheSeamau5/elm-check": "3.2.0 <= v < 4.0.0",
```

Comment:
Bumped to `3.2.0` which now has [lowerCaseChar and upperCaseChar](https://github.com/TheSeamau5/elm-check/blob/master/src/Check/Investigator.elm#L143-L155).